src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,13 @@
+$day-size: 100px;
+$day-in-row: 7;
+$gap-size: 1px;
+$YPaddings: 10px;
+$days-of-week:
+  "mon" 0,
+  "tue" 101px,
+  "wed" 202px,
+  "thu" 303px,
+  "fri" 404px,
+  "sat" 505px,
+  "sun" 606px;
```

Comment:
Do not hardcode pixel values. Create a loop and calculate these values in `calendar`.